specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2024-09-01/dataConnectorDefinitions.json

```diff
@@ -0,0 +1,706 @@
+        },
+        "permissions": {
+          "$ref": "#/definitions/ConnectorDefinitionsPermissions"
+        },
+        "instructionSteps": {
+          "description": "Gets or sets the instruction steps to enable the connector.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/InstructionStep"
+          },
+          "x-ms-identifiers": []
+        },
+        "logo": {
+          "description": "Gets or sets the connector logo to be used when displaying the connector within Azure Sentinel's connector's gallery.\r\nThe logo value should be in SVG format.",
+          "type": "string"
+        },
+        "isConnectivityCriteriasMatchSome": {
+          "description": "Gets or sets a value indicating whether to use 'OR'(SOME) or 'AND' between ConnectivityCriteria items.",
+          "type": "boolean"
+        }
+      }
+    },
+    "CustomPermissionDetails": {
+      "description": "The Custom permissions required for the connector.",
+      "required": [
+        "description",
+        "name"
+      ],
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Gets or sets the custom permissions name.",
+          "type": "string"
+        },
+        "description": {
+          "description": "Gets or sets the custom permissions description.",
+          "type": "string"
+        }
+      }
+    },
+    "GraphQuery": {
+      "description": "The graph query to show the volume of data arriving into the workspace over time.",
+      "required": [
+        "baseQuery",
+        "legend",
+        "metricName"
+      ],
+      "type": "object",
+      "properties": {
+        "metricName": {
+          "description": "Gets or sets the metric name that the query is checking. For example: 'Total data receive'.",
+          "type": "string"
+        },
+        "legend": {
+          "description": "Gets or sets the legend for the graph.",
+          "type": "string"
+        },
+        "baseQuery": {
+          "description": "Gets or sets the base query for the graph.\r\nThe base query is wrapped by Sentinel UI infra with a KQL query, that measures the volume over time.",
+          "type": "string"
+        }
+      }
+    },
+    "InstructionStep": {
+      "description": "Instruction steps to enable the connector.",
+      "type": "object",
+      "properties": {
+        "title": {
+          "description": "Gets or sets the instruction step title.",
+          "type": "string"
+        },
+        "description": {
+          "description": "Gets or sets the instruction step description.",
+          "type": "string"
+        },
+        "instructions": {
+          "description": "Gets or sets the instruction step details.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/InstructionStepDetails"
+          },
+          "x-ms-identifiers": []
+        },
+        "innerSteps": {
```

Comment:
>innerSteps [](http://example.com/codeflow?start=9&length=10)

Hard to see how this is different/better than just putting everything into one "instructions" list. How does a client combine the two lists?